pkg/clusteragent/admission/common/label_selectors.go

```diff
@@ -19,7 +19,11 @@ func DefaultLabelSelectors(useNamespaceSelector bool) (namespaceSelector, object
 
 	if pkgconfigsetup.Datadog().GetBool("admission_controller.mutate_unlabelled") ||
 		pkgconfigsetup.Datadog().GetBool("apm_config.instrumentation.enabled") ||
-		len(pkgconfigsetup.Datadog().GetStringSlice("apm_config.instrumentation.enabled_namespaces")) > 0 {
```

Comment:
A note for those reading this, this is used across all webhooks and isn't very optimized because of the dependencies across all of them (which we are trying to decouple).

@betterengineering Add a `TODO` here to create different label selectors and use them for the different webhooks (or make one webhook?). There are options. 

cc @davidor @adel121 this definitely needs a follow up from your team.